users/tests.py

```diff
@@ -1,3 +1,171 @@
 # from django.test import TestCase
+import pytest
+from users.models import Profile
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
+from django.db import IntegrityError
+from django.utils import timezone
 
-# Create your tests here.
+
+CREATION_DATE = timezone.now()
+
+
+USERNAME = "testuser"
+FIRSTNAME = "Test"
+LASTNAME = "User"
+PASSWORD = "testpass"
+EMAIL = "testuser@gmail.com"
+PHONE_NUMBER = "0541234567"
+LONG_PROFESSION = " test test test test test test test test test test test test test test test test" \
+                 " test test test test test test test test test" \
+                 " test test test test test test test test test" \
+                 " test test test test "
+LONG_BIO = """ test test test test test test test test test test test test test test test test
+                test test test test test test test test test test test test test test test test
+                test test test test test test test test test test test test test test test test
+                test test test test test test test test test test test test test test test test"""\
+                 """ test test test test test test test test test test test test test test test test
+                   test test test test test test test test test""" \
+                 " test test test test test test test test test" \
+                 " test test test test "
+
+
+@pytest.fixture
+def new_user():
+    user = User(username=USERNAME, first_name=FIRSTNAME, last_name=LASTNAME, email=EMAIL)
+    user.set_password(PASSWORD)
+    return user
+
+
+@pytest.fixture
+def new_second_user():
+    user = User(username='second_user', first_name='second', last_name='user', email='email@gmail.com')
+    user.set_password('second_pass')
+    return user
+
+
+@pytest.fixture
+def persist_user(new_user):
+    new_user.save()
+    return new_user
+
+
+@pytest.fixture
+def persist_second_user(new_second_user):
+    new_second_user.save()
+    return new_second_user
+
+
+@pytest.mark.django_db()
+class TestUserModel():
+    # Check if the user is saved in the database
+    def test_persist_user(self, new_user):
+        new_user.save()
+        assert new_user in User.objects.all()
+        assert new_user.profile in Profile.objects.all()
```

Comment:
no need to test dJango builtins, the dJango folks probably already tested this method to be working.

Reply:
I deleted this